MFVNeutralino/test/gen_histos.py

```diff
@@ -3,9 +3,16 @@
 
 debug = 'debug' in sys.argv
 
-sample_files(process, 'ggHToSSTodddd_tau1mm_M55_2017', 'ntupleulv4bm', 5)
+sample_files(process, 'mfv_stopld_tau001000um_M1000_2018', 'miniaod', 5)
+#sample_files(process, 'ggHToSSTodddd_tau1mm_M55_2017', 'ntupleulv4bm', 5)
 #sample_files(process, 'mfv_neu_tau001000um_M0300_2017', 'ntupleulv4bm', 5)
 tfileservice(process, 'gen_histos.root')
+#tfileservice(process, 'stopld_genhisto_M1000_1mm.root')
+#input_files(process, '/store/mc/RunIISummer20UL17MiniAODv2/ZH_HToSSTodddd_ZToLL_MH-125_MS-55_ctauS-1_TuneCP5_13TeV-powheg-pythia8/MINIAODSIM/106X_mc2017_realistic_v9-v2/2550000/13DF01B3-1BC9-0246-8C88-DF26E2F16793.root')
+#input_files(process, '/store/mc/RunIISummer20UL17MiniAODv2/WplusH_HToSSTodddd_WToLNu_MH-125_MS-55_ctauS-1_TuneCP5_13TeV-powheg-pythia8/MINIAODSIM/106X_mc2017_realistic_v9-v2/40000/0BD790C6-883F-0147-A66E-8EC9DC53750F.root')
+input_files(process, '/store/mc/RunIISummer20UL17MiniAODv2/WminusH_HToSSTodddd_WToLNu_MH-125_MS-55_ctauS-1_TuneCP5_13TeV-powheg-pythia8/MINIAODSIM/106X_mc2017_realistic_v9-v2/70000/1966857B-FA0C-454B-A180-F83F96A0C9D6.root')
+#input_files(process, './13DF01B3-1BC9-0246-8C88-DF26E2F16793.root')
+max_events(process, 100)
```

Comment:
What are we trying to do here? It looks like we're loading stop->ld and then overriding it to load WH->LLPs? Let's do some cleanup